### DISCUSSION TITLE
Simple Question (Im totally New)

### DISCUSSION THREAD

**Discussion by Ashinthya** [smartcontractkit/full-blockchain-solidity-course-py#1007] [Q&A] [answered]
![image_2022-02-10_132711](https://user-images.githubusercontent.com/88304999/153362547-8487b1c7-4edc-41d8-ae43-f8593a7b5c29.png)

Is that a normal Bracket or something else?

**Comment by cromewar** [comment, marked as answer]
Hello @Ashinthya 
I don't know what you mean with "normal" but as you are new on coding let me give you a hand explaining the syntax here

This is a function, a function is a portion of code contained on a space and it could be used as many times as you want, the function syntax is as follows 

`function name(type _varName) public {}`

where

function: is a keyword
name: is the name you want to give to the function
(): the brackets helps to know that is a function and also inside of them you set up the input parameters
type: the data type you want to use, for example uint, bool, string... etc.
_varName:  the name you want to give to that input variable.
public: is the visibility this could be private, public, internal and so on.

**Reply by Ashinthya** [reply]
oh Thank you very much ! It's great help

**Comment by n4n0b1t3** [comment]
Parentheses are smooth and curved ( ), brackets are square [ ], and braces are curly { }.

**Reply by Ashinthya** [reply]
Thankz, I called all of them brackets 🙏😂
